src/OMSimulatorLib/Flags.cpp

```diff
@@ -391,6 +396,32 @@ oms_status_enu_t oms::Flags::StartTime(const std::string& value)
   return oms_status_ok;
 }
 
+std::vector<std::string> split(const std::string& s, char delim)
+{
+  std::vector<std::string> result;
+  std::stringstream ss(s);
+  std::string item;
+
+  while (getline(ss, item, delim))
+    result.push_back(item);
+
+  return result;
+}
+
+oms_status_enu_t oms::Flags::StepSize(const std::string& value)
+{
+  std::vector<std::string> stepSize = split(value, ',');
+  if (stepSize.size() > 1)
+  {
+    GetInstance().initialStepSize = atof(stepSize[0].c_str());
```

Comment:
We could add some basic error checks, e.g. no negative numbers for steps sizes.
But that was the only way I could find to break this flag.

Reply:
done